gateway/src/apicast/policy/3scale_referrer/3scale_referrer.lua

```diff
@@ -0,0 +1,20 @@
+local policy = require('apicast.policy')
+local _M = policy.new('3scale Referrer policy')
```

Comment:
I added the 3scale prefix here because this is 3scale specific. Like the `3scale_batcher` policy.